README.md

```diff
@@ -30,5 +30,6 @@ After cloning, do `npm install` to set up your environment. You can then do `npm
 
 ## Community
 
-* [Es-discuss mailing list](https://esdiscuss.org): Mailing list for ECMAScript discussions
-* IRC: #tc39 on Freenode ([instructions](https://freenode.net/kb/answer/chat))
+* [ES discourse](https://es.discourse.group/): Discourse for ECMAScript discussion and questions.
+* [Es-discuss mailing list](https://esdiscuss.org): Mailing list for ECMAScript discussions.
```

Comment:
These two lines just got an added period

Reply:
imo it's weird for lists to have periods; they're not sentences. can we remove all three?